packages/styled-components/src/sheet/test/Rehydration.test.js

```diff
@@ -94,4 +100,19 @@ describe('rehydrateSheet', () => {
     expect(sheet.getTag().tag.length).toBe(0);
     expect(styleHead.parentElement).toBe(document.head);
   });
+
+  it('tolerates long, malformed CSS', () => {
+    document.head.innerHTML = `
+      <style ${SC_ATTR} ${SC_ATTR_VERSION}="${SC_VERSION}">
+        {xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+          xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+          }
+        .rule {}/*!sc*/
+        ${SC_ATTR}.g1[id="idA"]{content:""}/*!sc*/
```

Comment:
Maybe this test CSS should include a real inline comment with `/*!` wrapped around it, rather than only `sc`? What am I missing here?

Reply:
This is just the new "splitter" marker we are adding. You can consider it an implementation detail for SSR.